generate_workspace/src/main/java/com/google/devtools/build/workspace/maven/Resolver.java

```diff
@@ -321,6 +322,14 @@ private String downloadSha1(Rule rule) {
     return null;
   }
 
+  /**
+   * Prints currently downloaded maven artifact to console.
+   */
+  private static void logArtifactDownload(Model model) {
+    System.out.println("\tDownloading maven artifact " + model.getGroupId() + ":"
```

Comment:
That makes sense about the phrasing. As far as the logging, I was previously using it. However, I found it extremely verbose. For example, here is what it outputs for the above command:

```
____Running command line: bazel-bin/generate_workspace/generate_workspace '--artifact=org.powermock:powermock-module-junit4:1.5.1' '--artifact=com.airbnb.okreplay:gradle-plugin:1.1.0'
Jun 14, 2017 10:52:07 AM com.google.devtools.build.workspace.maven.Resolver resolveEffectiveModel
INFO: 	Downloading pom for org.powermock:powermock-module-junit4:1.5.1
Jun 14, 2017 10:52:07 AM com.google.devtools.build.workspace.maven.Resolver resolveEffectiveModel
INFO: 	Downloading pom for junit:junit:4.11
Jun 14, 2017 10:52:07 AM com.google.devtools.build.workspace.maven.Resolver resolveEffectiveModel
INFO: 	Downloading pom for org.hamcrest:hamcrest-core:1.3
Jun 14, 2017 10:52:07 AM com.google.devtools.build.workspace.maven.Resolver resolveEffectiveModel
INFO: 	Downloading pom for org.powermock:powermock-module-junit4-common:1.5.1
Jun 14, 2017 10:52:07 AM com.google.devtools.build.workspace.maven.Resolver resolveEffectiveModel
INFO: 	Downloading pom for org.powermock:powermock-core:1.5.1
Jun 14, 2017 10:52:07 AM com.google.devtools.build.workspace.maven.Resolver resolveEffectiveModel
INFO: 	Downloading pom for org.powermock:powermock-reflect:1.5.1
Jun 14, 2017 10:52:08 AM com.google.devtools.build.workspace.maven.Resolver resolveEffectiveModel
INFO: 	Downloading pom for org.objenesis:objenesis:1.2
Jun 14, 2017 10:52:08 AM com.google.devtools.build.workspace.maven.Resolver resolveEffectiveModel
INFO: 	Downloading pom for org.javassist:javassist:3.18.0-GA
Jun 14, 2017 10:52:08 AM com.google.devtools.build.workspace.maven.Resolver resolveEffectiveModel
INFO: 	Downloading pom for com.airbnb.okreplay:gradle-plugin:1.1.0
Jun 14, 2017 10:52:08 AM com.google.devtools.build.workspace.maven.Resolver resolveEffectiveModel
WARNING: Could not resolve dependency com.android.tools.build:gradle:2.3.1: Could not find any repositories that knew how to resolve com.android.tools.build:gradle:2.3.1 (checked https://repo1.maven.org/maven2/, https://oss.sonatype.org/content/repositories/snapshots)
Jun 14, 2017 10:52:08 AM com.google.devtools.build.workspace.maven.Resolver resolveEffectiveModel
INFO: 	Downloading pom for org.jetbrains.kotlin:kotlin-stdlib:1.1.2
Jun 14, 2017 10:52:08 AM com.google.devtools.build.workspace.maven.Resolver resolveEffectiveModel
INFO: 	Downloading pom for org.jetbrains:annotations:13.0
Jun 14, 2017 10:52:08 AM com.google.devtools.build.workspace.maven.Resolver resolveEffectiveModel
INFO: 	Downloading pom for com.android.tools.ddms:ddmlib:25.3.0
Wrote /private/var/tmp/_bazel_petroseskinder/f4ae163fe842998e2d78cf9f858a5836/execroot/migration-tooling/bazel-out/local-fastbuild/bin/generate_workspace/generate_workspace.runfiles/__main__/generate_workspace.bzl
```

I have no problems changing it. However, I do believe that we shouldn't output `Jun 14, 2017 10:52:08 AM com.google.devtools.build.workspace.maven.Resolver resolveEffectiveModel` for each pom file. 

Alternative ideas? I could just consolidate each of these log statements into a single statement.